lib/newman_conway.rb

```diff
@@ -1,7 +1,28 @@
 
 
-# Time complexity: ?
-# Space Complexity: ?
+# Time complexity: O(log n)
+# Space Complexity: O(n)
 def newman_conway(num)
```

Comment:
How could the time complexity be O(log n) you are looping count from 3 to num?

It's actually O(n) time complexity.